src/Snackbar/Snackbar.js

```diff
@@ -197,6 +202,8 @@ class Snackbar extends Component {
     }, 400);
   }
 
+  contentId = UniqueId('snackbar');
```

Comment:
That sound better. Also noticed that we want to focus on bug fixes for the `master` branch. Ongoing work are on the `next` branch. Please consider having a look at that version. We have made quite some improvement on the accessibility part.